File: backend/src/main/java/com/daromon/goodeventbackend/Security/AuthResponse.java
package com.daromon.goodeventbackend.Security;

import com.daromon.goodeventbackend.Event.Event;
import com.daromon.goodeventbackend.UserFavouriteEvent.UserFavorites;

import java.util.List;

public class AuthResponse {

    private String username;
    private String accessToken;
    private Long idUser;
    private List<UserFavorites> favoriteEvents;
    public AuthResponse(String username, String accessToken, Long idUser, List<UserFavorites> favoriteEvents) {
        this.username = username;
        this.accessToken = accessToken;
        this.idUser = idUser;
        this.favoriteEvents = favoriteEvents;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getAccessToken() {
        return accessToken;
    }

    public void setAccessToken(String accessToken) {
        this.accessToken = accessToken;
    }


    public Long getIdUser() {
        return idUser;
    }

    public void setIdUser(Long idUser) {
        this.idUser = idUser;
    }

    public List<UserFavorites> getFavoriteEvents() {
        return favoriteEvents;
    }

    public void setFavoriteEvents(List<UserFavorites> favoriteEvents) {
        this.favoriteEvents = favoriteEvents;
    }
}

File: frontend/app/src/main/java/daromon/goodevent/retrofit/AuthResponse.kt
package daromon.goodevent.retrofit

import daromon.goodevent.model.EventData

class AuthResponse {
    private var username: String? = null
    private var accessToken: String? = null
    private var idUser: Long? = null


    fun AuthResponse(username: String?, accessToken: String?, userId: Long ) {
        this.username = username
        this.accessToken = accessToken
        this.idUser=userId

    }

    fun getUsername(): String? {
        return username
    }

    fun setUsername(username: String?) {
        this.username = username
    }

    fun getAccessToken(): String? {
        return accessToken
    }

    fun setAccessToken(accessToken: String?) {
        this.accessToken = accessToken
    }
    fun getUserId() : Long?
    {
        return this.idUser
    }



}